src/components/atoms/UiIcon/UiIcon.stories.ts

```diff
@@ -0,0 +1,179 @@
+import type {
+  Meta,
+  StoryObj,
+} from '@storybook/vue3';
+import { UiIcon } from '@/../index';
+import { defineAsyncComponent } from 'vue';
+import icons from './icons';
+import './UiIcon.stories.scss';
+
+const meta = {
+  title: 'Atoms/Icon',
+  component: UiIcon,
+  argTypes: {
+    icon: {
+      control: { type: 'select' },
+      options: icons,
+    },
+  },
+} satisfies Meta<typeof UiIcon>;
+export default meta;
+type Story = StoryObj<typeof UiIcon>;
+export const Basic: Story = {
+  render: (args) => ({
+    components: { UiIcon },
+    setup() {
+      return { args };
+    },
+    template: '<UiIcon v-bind="args"/>',
+  }),
+};
+Basic.args = { icon: 'absent' };
+Basic.parameters = {
+  docs: {
+    source: {
+      code: `<template>
+    <UiIcon :icon="icon" />
+</template>
+
+<script lang="ts">
+import { ref } from 'vue';
+import { UiIcon } from '@infermedica/component-library';
+
+const icon = ref('absent');
+</script>`,
+    },
+  },
+};
+export const CustomIcon: Story = {
+  render: () => ({
+    components: { UiIcon },
+    setup() {
+      const icon = defineAsyncComponent(() => import('../../../../public/logo.svg'));
+      return { icon };
+    },
+    template: `<UiIcon 
+      :icon="icon"
+      class="custom-icon"
+    />`,
+  }),
+};
+CustomIcon.parameters = {
+  docs: {
+    source: {
+      code: `<template>
+  <UiIcon 
+    :icon="icon"
+    class="custom-icon" 
+  />
+</template>
+
+<script lang="ts">
+import { defineAsyncComponent } from 'vue';
+import { UiIcon } from '@infermedica/component-library';
+
+const icon = defineAsyncComponent(() => import('../../../assets/icons/ce.svg'));
+</script>`,
+    },
+  },
+};
+export const AllIcons: Story = {
+  render: (args) => ({
+    components: { UiIcon },
+    setup() {
+      const { icons } = args;
+      const handleIconClick = (icon: string) => {
+        navigator.clipboard.writeText(icon);
+      };
+      return { icons };
+    },
+    template: `<template 
+      v-for="(icon, key) in icons" 
+      :key="key"
+    >
+      <UiIcon 
+          :icon="icon"
+          @click="handleIconClick(icon)"
+      />
+    </template>`,
+  }),
+};
+AllIcons.args = {
+  icons: icons.filter((icon) => {
+    const illustrations = [
+      'agreement',
+      'agreement-rtl',
+      'boy',
+      'boy-rtl',
+      'lock',
+      'no-internet-illustration',
+      'no-internet-illustration-rtl',
+      'podium',
+      'podium-rtl',
+    ];
+    return !illustrations.includes(icon);
+  }),
+};
+export const IllustrationWithCustomColors: Story = { ...Basic };
+IllustrationWithCustomColors.args = {
+  icon: 'boy',
+  class: [
+    'icon-as-illustration',
+    'icon-as-custom-illustration',
+  ],
+};
+IllustrationWithCustomColors.parameters = {
+  docs: {
+    source: {
+      code: `<template>
```

Comment:
<img width="1014" alt="Zrzut ekranu 2023-03-13 o 21 20 11" src="https://user-images.githubusercontent.com/12138170/224823207-c05b480e-10a5-461f-a479-789607bc49df.png">
I love that; Everything in one place; Instead that
![Uploading Zrzut ekranu 2023-03-13 o 21.22.03.png…]()